### ISSUE TITLE
Getting exceptions after upgrading to arc 2.8.0

### ISSUE THREAD

**Issue by aissar** [tripl-ai/docker#3]
Hey guys,

I'm getting the following errors when submitting arc jobs using 2.80. The same job runs under 2.70 with no issues. Arc job attached for reference.

**Host machine environment**

NAME="Red Hat Enterprise Linux Server"
VERSION="7.7 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.7"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.7 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.7:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.7
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.7"

**Working docker image**

arc_2.7.0_spark_2.4.4_scala_2.11_hadoop_2.7.7_1.0.0

**Docker image with exception**

arc_2.8.0_spark_2.4.5_scala_2.11_hadoop_2.7.7_1.0.0 - see attachment 1
arc_2.8.0_spark_2.4.5_scala_2.12_hadoop_2.9.2_1.0.0 - see attachment 2

**Attachments**

<img width="1191" alt="arc_2 8 0_spark_2 4 5_scala_2 11_hadoop_2 7 7_1 0 0" src="https://user-images.githubusercontent.com/37895090/77304331-6652d980-6d48-11ea-887b-a018d6d66a11.png">
<img width="1187" alt="arc_2 8 0_spark_2 4 5_scala_2 12_hadoop_2 9 2_1 0 0" src="https://user-images.githubusercontent.com/37895090/77304352-6c48ba80-6d48-11ea-87a7-cf2217abca2d.png">

[etl_job_v0.2.json.zip](https://github.com/tripl-ai/arc/files/4368388/etl_job_v0.2.json.zip)

**Comment by aissar** [comment]
Hi @seddonm1 confirming I no longer get these exceptions when using the following image. Please feel free to close the issue once this fixed is merged into the official repo.

seddonm1/arc:arc_2.8.1_spark_2.4.5_scala_2.12_hadoop_2.9.2_1.0.0

**Comment by seddonm1** [comment]
FYI resolved with 2.9.0+
